mangum/response.py

```diff
@@ -0,0 +1,9 @@
+from dataclasses import dataclass
+from typing import List
+
+
+@dataclass
+class Response:
```

Comment:
Maybe a new `datastructures.py` file that combines the classes in `response.py` and `scope.py` in a single file.

Reply:
I re-worked the __init__ files a bit to cleanup imports